packages/signature-verifier-api/src/msg.rs

```diff
@@ -0,0 +1,15 @@
+use cosmwasm_schema::{cw_serde, QueryResponses};
+use cosmwasm_std::{HexBinary, Uint64};
+
+#[cw_serde]
+#[derive(QueryResponses)]
+pub enum QueryMsg {
+    #[returns(bool)]
+    VerifySignature {
+        signature: HexBinary,
+        message: HexBinary,
+        public_key: HexBinary,
+        signer_address: String,
+        session_id: Uint64,
+    },
+}
```

Comment:
I would actually rather have this live in the msg.rs file of the multisig contract, so that way it's clearer to readers the interface of the callback. Right now, when you read the msg.rs file of multisig, there is mention of a callback, but no indication about what the interface to that callback is. We could leave a comment, but I feel just having this type right there would be preferable.

Reply:
I don't know, I think the documentation should show the proper package instead of having everything in the multisig. Seems more cleaner to me if the implementations just import the interface package